README.md

```diff
@@ -105,7 +105,8 @@ A few things to mention about the contents of the repository:
 * [audio_classify.zip](audioset_classify.zip): This is the core training code we will be running on IBM Cloud. 
 * [audioset_classify](audioset_classify/): The training code for reference. We will use the first .zip file to upload this code to the cloud.
 * [training-runs.yml](training-runs.yml) This file is required to perform the training on IBM Cloud. It is used to setup training metadata and connection information.
-* [audioclassify_inference.ipynb](audioclassify_inference.ipynb) This is the notebook we will be using to perform inference after training. 
+* [audioclassify_inference.ipynb](audioclassify_inference.ipynb) This is the notebook we will be using to perform inference after training.
+* In addition to these files, you will need to download two more files: [eval_segments.csv](https://research.google.com/audioset/download.html) from the the [Audioset dataset page](https://research.google.com/audioset/download.html) and [class_label_indices.csv](https://github.com/IBM/MAX-Audio-Classifier/blob/master/samples/class_labels_indices.csv) from the MAX Audio Classifier [GitHub repo](https://github.com/IBM/MAX-Audio-Classifier/blob/master/samples/class_labels_indices.csv). We use this to visualize some results in the inference notebook.
```

Comment:
```
 you will need to download two more files: [eval_segments.csv](https://research.google.com/audioset/download.html) from the the [Audioset dataset page](https://research.google.com/audioset/download.html)
```

Should the first URL be different? There seems little value on linking to the same URL twice.

Reply:
2. 

```
and [class_label_indices.csv](https://github.com/IBM/MAX-Audio-Classifier/blob/master/samples/class_labels_indices.csv) from the MAX Audio Classifier [GitHub repo](https://github.com/IBM/MAX-Audio-Classifier/blob/master/samples/class_labels_indices.csv)
```

This PR deletes the referenced file

Reply:
for 1 -> I'll remove one of the URLs.

for 2 -> I removed the CSV files from this repo because they are under different licenses so trying to just link it directly from the source.